x/dymns/utils/chain.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import "regexp"
+
+// patternValidChainId is a regex pattern for valid chain id format.
+var patternValidChainId = regexp.MustCompile(`^[a-z]+(-[a-z]+)?(_\d+)?(-\d+)?$`)
+
+// IsValidChainIdFormat returns true if the given string is a valid chain id format.
+// It checks the length and the pattern of the chain id.
+// The chain id length can be between 3 and 47 characters.
+func IsValidChainIdFormat(chainId string) bool {
+	if len(chainId) < 3 || len(chainId) > 47 {
```

Comment:
can you point out where it is?

Reply:
No, I mean we should add it to that library, so it can be reused.

Also, for the max chain ID length, you can consider using [this](https://github.com/cometbft/cometbft/blob/76809d58c8ce201c413c603097c3c5b715448f34/types/genesis.go#L21)

Reply:
ok I added a TODO for moving that, I won't do it in this PR because I don't want to wait for merge or touching the go.mod, very annoying with rebase